packages/ui/src/PasswordField.tsx

```diff
@@ -22,6 +22,7 @@ import styles from './PasswordField.module.scss'
 
 type PasswordFieldCoreProps = {
   name: string
+  id?: string
```

Comment:
`id` is automatically generated by the `TextField`. You'll have to propagate it there as well.
BTW, should it go on L42 instead?